testdata/json_test_schema.json.golden

```diff
@@ -67,6 +67,7 @@
       "virtual": false
     }
   ],
+  "subroutines": null,
```

Comment:
Compared to https://github.com/k1LoW/tbls/pull/341/files#diff-7dde8826bb86aa9a1addcc1aedcda0cb76b0b57e2d20d5415cf8fd5ac6baa3b6R49, there seems to be a mixture of null and [], do you know why?

Reply:
fixed in [f0a2289](https://github.com/k1LoW/tbls/pull/341/commits/f0a2289b0c6398cb58c6e3f9e09b057abbc14cea). Is it ok to have empty array?